### ISSUE TITLE
Can't run the project

### ISSUE THREAD

**Issue by alejandronanez** [gitpoint/git-point#29]
I follow the steps under CONTRIBUTING.md but couldn't run the project. What I'm seeing is this:
 ```
No bundle URL present.
Make sure you're running a package server or have included a .jsbundle file in your application bundle
```
Any idea what could be wrong?

Thanks!

**Comment by housseindjirdeh** [comment]
Aaaah that's most likely due to me removing a localhost exception in `Info.plist` since I had to in order to deploy the app. Included it back in (https://github.com/gitpoint/git-point/commit/f471f044ecf60209da62be1a84d96e9979bcf93a) - I'll remember to remove it locally before I deploy the next update.

**Comment by machour** [comment]
Facing the exact same error message today.

Reproduce case:
```
git clone git@github.com:gitpoint/git-point.git git-point-orig
cd git-point-orig
yarn && yarn run link && yarn start:ios
``` 
None of the provided troubleshooting options provided in `CONTRIBUTING.md` did the trick for me.

The localhost exception **is** present in `Info.plist`

Environment infos:
```
🍺  ~/git-point-orig (master)*$ node --version
v8.1.3
🍺  ~/git-point-orig (master)*$ npm --version
5.0.3
🍺  ~/git-point-orig (master)*$ yarn --version
0.27.5
🍺  ~/git-point-orig (master)*$ react-native --version
react-native-cli: 2.0.1
react-native: 0.46.2
The app is loaded in a simulator with the following specs: iPhone 6 - iOS 10.3
Running all this on a MacBook Pro (15-inch, 2016) with OsX 10.12.6
```

Let me know if I can provide more details

**Comment by housseindjirdeh** [comment]
@machour are you still having trouble with iOS and have only been able to test on Android? Please let me know and we'll reopen this. Apologies if we didn't get to the bottom of this in our gitter chat but I can't recall if we have :thinking:

**Comment by machour** [comment]
Just did a fresh checkout and hit the same issue again: "No bundle URL is present".
This is the case when running "yarn start:ios" or "yarn start -- --reset-cache" and then yarn run ios in a another terminal window

```
🍺  ~/git-point (master)*$ yarn --version
0.27.5
🍺  ~/git-point (master)*$ npm --version
5.3.0
🍺  ~/git-point (master)*$ node --version
v8.4.0
```

**Comment by machour** [comment]
I did some further digging, starting with a fresh repo and XCode, here's what I've noticed:

- when running `yarn start:ios`, the `ios/GitPoint/main.jsbundle` file doesn't get generated. Is that normal ?
- I'm able to generate that file using `react-native bundle --entry-file='index.ios.js' --bundle-output='./ios/GitPoint/main.jsbundle' --dev=false --platform='ios' --assets-dest='./ios'`, but I'm not sure it's needed
- I'm able to run the app on the iOS Simulator by hardcoding the URL and building from XCode:
```diff
--- AppDelegate.m	2017-09-08 13:14:40.000000000 +0100
+++ AppDelegate.patched	2017-09-08 13:14:34.000000000 +0100
@@ -22,7 +22,7 @@


 #ifdef DEBUG
-  jsCodeLocation = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index.ios" fallbackResource:nil];
+  jsCodeLocation = [NSURL URLWithString:@"http://192.168.1.2:8081/index.ios.bundle"];
 #else
   jsCodeLocation = [CodePush bundleURL];
 #endif
```
- with that hardcoded url, and toggling "Enable live reload" everything works as expected
